presto-druid/src/main/java/com/facebook/presto/druid/DruidBrokerPageSource.java

```diff
@@ -144,11 +143,12 @@ else if (type instanceof RealType) {
                             type.writeLong(blockBuilder, floatToRawIntBits(value.floatValue()));
                         }
                         else if (type instanceof TimestampType) {
-                            DateTimeFormatter formatter = ISODateTimeFormat.dateTimeParser()
-                                    .withChronology(ISOChronology.getInstanceUTC())
-                                    .withOffsetParsed();
-                            DateTime dateTime = formatter.parseDateTime(value.textValue());
-                            type.writeLong(blockBuilder, dateTime.getMillis());
+                            DateTimeFormatter formatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME
+                                    .withChronology(IsoChronology.INSTANCE);
+
+                            OffsetDateTime dateTime = OffsetDateTime
+                                    .parse(value.textValue(), formatter);
```

Comment:
@zhenxiao Hi Zhenxiao, we are trying to replace Joda time with Java time. However, it seems like there're some time formats that Joda.time can parse but the java.time cannot. Do you have an idea what input format of the time string (`value.textValue()`) is receiving? For example, the parse method of OffsetDateTime is expecting an input string in the format of 2007-12-03T10:15:30+01:00.